src/test/order_summary/OrderSummaryController.integration.test.ts

```diff
@@ -87,11 +93,13 @@ describe("OrderSummaryController", () => {
             chai.expect($("#delivery-address-value").length).to.equal(0);
             chai.expect($("#subtotal-list").text()).to.contain("1234567");
             chai.expect($("#subtotal-list").text()).to.contain("£15");
+            chai.expect(getBasketStub).to.have.been.called;
```

Comment:
Did you try ` chai.expect(response.text).to.contain( "Basket (1)" );` here?

Reply:
Added on here now